e2e/test/scenarios/visualizations/object_detail.cy.spec.js

```diff
@@ -259,6 +259,34 @@ describe("scenarios > question > object details", () => {
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
     cy.findByText(/Item 1 of/i).should("be.visible");
   });
+
```

Comment:
Wrote an e2e test but I'm not sure how necessary this one is, considering there is an extremely similar test in `ObjectDetailWrapper.unit.spec.tsx`. My reasoning for this test to exist would be to test the behavior in a more "real-world" environment but it also may be overkill